ovh/provider.go

```diff
@@ -23,8 +18,7 @@ func Provider() *schema.Provider {
 		Schema: map[string]*schema.Schema{
 			"endpoint": {
 				Type:        schema.TypeString,
-				Required:    true,
-				DefaultFunc: schema.EnvDefaultFunc("OVH_ENDPOINT", nil),
```

Comment:
Should we keep this one (as we are keeping it on application_key etc...) ?

Reply:
You're right, I'll delete all the `EnvDefaultFunc`